client/src/components/start/card-section/CardItem.tsx

```diff
@@ -0,0 +1,37 @@
+"use client";
+import { Card, CardDescription, CardTitle } from "@/components/ui/card";
+import Image from "next/image";
+import { useState } from "react";
+import { ICard } from "@/interfaces/start-page/card";
+
+interface CardSectionProps {
+  card: ICard;
+}
+const fallbackImage = "/images/start-page/fallback-image.jpg";
+
+export const CardItem = ({ card }: CardSectionProps) => {
+  const [imgSrc, setImgSrc] = useState(card.imageSrc || fallbackImage);
+
+  return (
+    <div className="flex flex-1 cursor-pointer flex-col transition-all delay-150 duration-300 md:hover:scale-110">
```

Comment:
I don't think any of these cards need a pointer cursor, because they are not clickable

Reply:
ok